### PR TITLE
feat: display the auth secret when creating an environment

### PR DESCRIPTION
**Pull request by slimee**
### General

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [x] Test manually the implemented changes
- [x] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [x] Consider the security impact of the changes made

### CODE REVIEW

File: test/commands/test-cli-helper/test-cli-context.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ const preparePlan = ({
   env,
   prompts,
   tokenBehavior,
+  additionnalStep,
```
Comment (slimee) [inline test/commands/test-cli-helper/test-cli-context.js:74]: the change in this file is to support the `additionnalStep` keyword, which adds a step at the end of the plan.

File: test/commands/test-cli-helper/test-cli-errors.js [hunk 2]
```diff
@@ -59,7 +59,9 @@ function errorIfStdRest(stds) {
   }
 }
 
-function errorIfBadCommand({ commandClass, commandArgs, commandPlan }) {
+function errorIfBadCommand({
```
Comment (slimee) [inline test/commands/test-cli-helper/test-cli-errors.js:62]: the diff in this file is to support the 'additionnalStep' keyword, which adds a step at the end of the plan.

**Review by jeffladiray — COMMENTED** [review]
LGTM, but 
image
`FOREST_AUTH_SECRET` seems a bit long compared to the one generated on the frontend. 

That's not an issue, but for consistency, I would suggest to reduce the length to 48 (Just like the frontend does)

File: src/services/environment-manager.js [hunk 3]
```diff
@@ -36,7 +38,7 @@ function EnvironmentManager(config) {
   this.createEnvironment = async () => {
     const authToken = authenticator.getAuthToken();
 
-    return agent
+    const environment = await agent
```
[the remark this answers is not in the document]
Comment (slimee) [reply src/services/environment-manager.js]: why not!

**Review by jeffladiray — APPROVED** [review]
Look good, works as expected.

Feel free to ignore my comment if you don't find it useful, maybe that's just me :D

**Pull request merged**